ui/src/app/cohort-search/attributes-page/attributes-page.component.ts

```diff
@@ -106,6 +107,7 @@ export class AttributesPageComponent implements OnDestroy, OnInit {
   }
 
   selectChange(index: number, option: any) {
+    this.selectedCode = option.code;
```

Comment:
If the attribute is for Blood Pressure Detail (subtype is 'BP') this should get the code for both select options and add them together